CHANGES.rst

```diff
@@ -6,8 +6,8 @@ Release history
 
 .. currentmodule:: skrub
 
-Ongoing development
-===================
+Release 0.5.0
+=============
```

Comment:
can you remove the paragraph below?

(actually I think we can also remove it from main when doing the post-release PR)

LGTM otherwise

Reply:
also you can remove the empty sections bug fixes and maintenance